app/(api)/api/candidate/route.ts

```
@@ -164,14 +168,17 @@ export async function POST(request: Request) {
     body.status = "APPROVED" as SubmissionStatus;
     body.slug = uniqueSlug;
-    body.electionId = electionId || null;
     body.verified = true;
     body.website = website || null;
     body.linkedin = linkedin || null;
-    body.additionalNotes = additionalNotes || null;
     body.hidden = true;
+    body.currentCity = city;
+    body.currentState = state;
+    // remove the city and state from the body
+    delete body.city;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)